function-web/src/main/java/io/micronaut/function/web/AnnotatedFunctionRouteBuilder.java

```diff
@@ -169,13 +174,8 @@ public void process(BeanDefinition<?> beanDefinition, ExecutableMethod<?, ?> met
                         LOG.debug("Created Route to Function: {}", route);
                     }
 
-                    if (consumes != null) {
```

Comment:
`consumes` and `produces` are never null, I have removed all the null checks and added length check